go.mod

```diff
@@ -1,6 +1,6 @@
 module go.mongodb.org/atlas-sdk/v20240805004
 
-go 1.22.0
+go 1.23
```

Comment:
1.23 is not a valid version see https://github.com/golang/go/issues/62278

Reply:
oh, i see, that's why I was asking AKO https://mongodb.slack.com/archives/C04N6N6KJUW/p1727769055935239